pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -764,10 +766,11 @@ default void delete(String topic, boolean force) throws PulsarAdminException {
      * @throws PulsarAdminException
      *             Unexpected error
      */
-    TopicStats getStats(String topic, boolean getPreciseBacklog) throws PulsarAdminException;
+    TopicStats getStats(String topic, boolean getPreciseBacklog,
```

Comment:
I think we shouldn't change the original API, that's will bring the compatibility issue with the PulsarAdmin. Let's create a new API for this.

Reply:
@zymap 
From what I've seen, this is only used by Admin CLI ([CmdTopics.java](https://github.com/apache/pulsar/blob/master/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java#L483-L490)), which provides default value <false>. for all needed parameters, e.g. getPreciseBacklog and subscriptionBacklogSize. And there's another default implementation of [TopicStats getStats(String topic)](https://github.com/apache/pulsar/blob/master/pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/Topics.java#L769-L771) which is used in some other places, it also has default value <false> provided for getPreciseBacklog and subscriptionBacklogSize. 
So it shouldn't create compatibility issue for users.

Reply:
I am thinking this is a user-faced API, user can call it from the `PulsarAdmin`. This change might be breaking their code.

Reply:
Make sense, I'll fix it.

Reply:
@zymap added a default `default TopicStats getStats(String topic, boolean getPreciseBacklog)` to make it backward compatible. And existing customer rely on this API won't be affected.